lineage/CellVar.py

```diff
@@ -32,7 +34,7 @@ def __init__(self, parent, gen, **kwargs):
             self.right = kwargs.get("right", None)
             self.obs = kwargs.get("obs", [])
 
-    def divide(self, T):
+    def divide(self, T: np.ndarray):
```

Comment:
```suggestion
    def divide(self, T: np.ndarray[float]):
```

Reply:
It says `Type object is not subscriptable`..

Reply:
Ok, ignore that then.